contracts/Compass.vy

```diff
@@ -219,3 +253,40 @@ def deploy_erc20(_paloma_denom: String[64], _name: String[64], _symbol: String[3
     event_id: uint256 = unsafe_add(self.last_event_id, 1)
     self.last_event_id = event_id
     log ERC20DeployedEvent(_paloma_denom, erc20, _name, _symbol, _decimals, event_id)
+
+@external
+def update_treasury(consensus: Consensus, _feeset_id: uint256, _community_fee: uint256, _security_fee: uint256, _community_wallet: address, _security_wallet: address):
```

Comment:
1. At the moment, the wallet addresses are not part of the governance. We could add them, but would it not be easier to have them be part of compass directly? Or would that be too expensive?

2. What do we do when deploying a new compass. Would it be enough to call `update_treasury` again to the new compass address?

Reply:
If we don't need to update it, we can set them as constant and it will save so much gas. And we can set them on deployment.